qiime2/core/transform.py

```diff
@@ -63,9 +63,9 @@ def make_transformation(self, other, recorder=None):
                      input_record=self._record, output_name=other._view_name,
                      output_record=other._record)
 
-        def transformation(view, validate_level='min'):
+        def transformation(view, level='min'):
```

Comment:
Revert these back as well (there's no "level" of transformation, rather we run the validation system, so the call-site would be ambiguous if called via key-word).